pipelines/_ingredients/MetricsForFakes.yaml

```diff
@@ -1,62 +0,0 @@
-# Miscellaneous metrics for Alert Production
-# In the future, these might be placed in task-specific pipelines (for debugging)
-# or grouped by their datasets (to optimize expensive Butler reads)
-
-description: Miscelaneous AP Pipeline metrics
-tasks:
-  apFakesCompletenessMag20t22:
-    class: lsst.ap.pipe.metrics.ApFakesCompletenessMetricTask
-    config:
-      connections.metric: ApFakesCompletenessMag20t22
-      magMin: 20
-      magMax: 22
-      connections.coaddName: parameters.coaddName
-      connections.fakesType: parameters.fakesType
-  apFakesCompletenessMag22t24:
-    class: lsst.ap.pipe.metrics.ApFakesCompletenessMetricTask
-    config:
-      connections.metric: ApFakesCompletenessMag22t24
-      magMin: 22
-      magMax: 24
-      connections.coaddName: parameters.coaddName
-      connections.fakesType: parameters.fakesType
-  apFakesCompletenessMag24t26:
-    class: lsst.ap.pipe.metrics.ApFakesCompletenessMetricTask
-    config:
-      connections.metric: ApFakesCompletenessMag24t26
-      magMin: 24
-      magMax: 26
-      connections.coaddName: parameters.coaddName
-      connections.fakesType: parameters.fakesType
-  apFakesCountMag20t22:
-    class: lsst.ap.pipe.metrics.ApFakesCountMetricTask
-    config:
-      connections.metric: ApFakesCountMag20t22
-      magMin: 20
-      magMax: 22
-      connections.coaddName: parameters.coaddName
-      connections.fakesType: parameters.fakesType
-  apFakesCountMag22t24:
-    class: lsst.ap.pipe.metrics.ApFakesCountMetricTask
-    config:
-      connections.metric: ApFakesCountMag22t24
-      magMin: 22
-      magMax: 24
-      connections.coaddName: parameters.coaddName
-      connections.fakesType: parameters.fakesType
-  apFakesCountMag24t26:
-    class: lsst.ap.pipe.metrics.ApFakesCountMetricTask
-    config:
-      connections.metric: ApFakesCountMag24t26
-      magMin: 24
-      magMax: 26
-      connections.coaddName: parameters.coaddName
-      connections.fakesType: parameters.fakesType
-  apFakesCount:
-    class: lsst.ap.pipe.metrics.ApFakesCountMetricTask
-    config:
-      connections.metric: ApFakesCount
-      magMin: 1
-      magMax: 39
-      connections.coaddName: parameters.coaddName
-      connections.fakesType: parameters.fakesType
```

Comment:
Did you mean to remove this task as well? I don't think we ever used it, but I don't see a counterpart in lsst/analysis_tools#286.